test/deviceid.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const i2c = require('../'),
+  i2c1 = i2c.openSync(42);
```

Comment:
Out of curiosity, what type of board was this tested on?
(The board has device `/dev/i2c-42`)

Reply:
raspbian stretch docker on pi3b with —device /dev/i2c-1:/dev/i2c-42